src/main/java/com/systelab/seed/features/patient/model/Patient.java

```diff
@@ -39,13 +37,11 @@ public class Patient extends ModelBase {
 
     private String email;
 
-    @Schema(description = "Date of Birth", example = "1966-11-17")
     private LocalDate dob;
 
```

Comment:
There are still some @Size and @NotNull annotations on this class that shouldn't be required as the validation responsibility has been transferred to the DTO